operator/pkg/helmreconciler/prune.go

```diff
@@ -143,6 +143,18 @@ func (h *HelmReconciler) PruneControlPlaneByRevisionWithController(iopSpec *v1al
 			return errStatus, err
 		}
 		allUslist = append(allUslist, uslist...)
+
+		crHash, err := h.getCRHash(c)
+		if err != nil {
+			return errStatus, err
+		}
+		scope.Infof("reset cache: %s for CR %s", c, crHash)
+		objCache := cache.GetCache(crHash)
+		if len(objCache.Cache) > 0 {
+			objCache.Mu.Lock()
+			objCache.Cache = map[string]*object.K8sObject{}
```

Comment:
Hi, one question here, why do we set an empty object here instead of directly setting it to nil?

Reply:
Just initialize outside `cache.GetCache`, if it is set to nil, then it will be initialized in the function